### PR TITLE
[ADM-402]  Refactor sourceControl store

### PR DESCRIPTION
**Pull request by mengna-tw**
## Changes
- refactor sourceControl store

### CODE REVIEW

File: frontend/src/context/config/sourceControl/sourceControlSlice.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    token: '',
+  },
+  isSourceControlVerified: false,
+}
+
+export const sourceControlSlice = createSlice({
+  name: 'sourceControl',
+  initialState: initialSourceControlState,
+  reducers: {
+    updateSourceControlVerifyState: (state, action) => {
```
Comment (guzhongren) [inline frontend/src/context/config/sourceControl/sourceControlSlice.tsx:22]: It's better to rename to updateVerifyState


And change others

**Review by guzhongren — COMMENTED** [review]
Good job

**Review by tingwang1234 — APPROVED** [review]
LGTM

File: frontend/src/context/config/sourceControl/sourceControlSlice.tsx (hunk 1 above)
Comment (mengna-tw) [reply frontend/src/context/config/sourceControl/sourceControlSlice.tsx:22]: Good point!
but there are three different verifyStates, includes updateBoardVerifyState, updatePipelineToolVerifyState, and updateSourceControlVerifyState,  so maybe keep current name is ok /  make sense

**Pull request merged**